cfgov/cfgov/settings/base.py

```diff
@@ -1,11 +1,14 @@
 import os
 from unipath import Path
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-PROJECT_ROOT = os.path.dirname(BASE_DIR)
-SECRET_KEY = os.environ.get('SECRET_KEY', os.urandom(32))
-V1_TEMPLATE_ROOT = PROJECT_ROOT + '/v1/jinja2/v1'
+# Repository root is 4 levels above this file
+REPOSITORY_ROOT = Path(__file__).ancestor(4)
+
+# This is the root of the Django project, 'cfgov'
+PROJECT_ROOT = REPOSITORY_ROOT.child('cfgov')
+V1_TEMPLATE_ROOT = PROJECT_ROOT.child('jinja2', 'v1')
```

Comment:
I'm pretty sure the `V1_TEMPLATE_ROOT` is actually being set to `cfgov-refresh/cfgov/jinja2/v1` where it should be `cfgov-refresh/cfgov/v1/jinja2/v1`